zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -931,17 +931,17 @@ brk.dialogue.confirmDropMessage.button.confirm.label = Drop
 brk.dialogue.confirmDropMessage.button.cancel.label = Cancel
 brk.name = Breakpoint Extension
 brk.optionspanel.name             = Breakpoints
-brk.optionspanel.option.alwaysOnTop.label	= ZAP always on top when breakpoint hit
+brk.optionspanel.option.alwaysOnTop.label	= ZAP Always on Top When Breakpoint Hit
```

Comment:
I'd suggest leaving this one and adding a period.

Reply:
Thanks. Updated commit and implemented suggestion.